src/locales/_template.json

```
@@ -5,7 +5,8 @@
     "home": "",
     "datasets": "",
     "services": "",
-    "data": "",
+    "external-data-link": "",
+    "license": "",
```

```suggestion
    "data": "",
```

The `data` key is used in the station information tab to open the timeseries plot